### PR TITLE
Added it all back

### PR DESCRIPTION
**Pull request by nabobalis**
Fixes https://github.com/sunpy/sunpy.org/issues/187

### CODE REVIEW

**Review by dstansby — COMMENTED** [review]
I remain unconvinced that it's worth us having subpackage maintainers (and in practice we don't), but we should special case coordinates and Albert somehow.

File: project/index.rst [hunk 1]
```diff
@@ -208,6 +208,57 @@ See :ref:`role_descriptions` for more details on each role.
     Previously, he was a postdoc at the Naval Research Laboratory and the Lockheed Martin Solar and Astrophysics Laboratory.
     In May of 2019, Will completed his PhD in the Department of Physics and Astronomy at Rice University under the direction of Stephen Bradshaw.
 
+``sunpy`` Subpackage Maintainers
```
Comment (Cadair) [inline project/index.rst:211]: Why don't we make this a more generic "maintainers" list, and include the names of people who maintain part of the whole ecosystem. I.e. include people who lead maintenance of sponsored packages? I feel like Danny and I do for ndcube what Albert does for coordinates?

Comment (dstansby) [reply project/index.rst:211]: I like this idea

Comment (nabobalis) [reply project/index.rst:211]: I can do that?

Comment (nabobalis) [reply project/index.rst]: have a look

File: project/index.rst [hunk 2]
```diff
@@ -208,6 +208,17 @@ See :ref:`role_descriptions` for more details on each role.
...
+Maintainers
+^^^^^^^^^^^
+
+The following section list the people who maintain various aspects of the SunPy Project that are not covered by the above roles.
+
+* sunpy.coordinates
+  `Albert Shih`_
+* ndcube
+  `Daniel Ryan`
+  `Stuart Mumford`
```
Comment (Cadair) [inline project/index.rst:220]: I would say that some or all of:

* sunxspex
* radiospectra
* sunpy-soar
* ablog?!

Could have people called out here?

File: project/index.rst [hunk 3]
```diff
@@ -208,6 +208,18 @@ See :ref:`role_descriptions` for more details on each role.
     Previously, he was a postdoc at the Naval Research Laboratory and the Lockheed Martin Solar and Astrophysics Laboratory.
     In May of 2019, Will completed his PhD in the Department of Physics and Astronomy at Rice University under the direction of Stephen Bradshaw.
 
+Maintainers
+^^^^^^^^^^^
+
+The following section list the people who maintain various aspects of the SunPy Project that are not covered by the above roles.
```
Comment (Cadair) [inline project/index.rst:214]: Do we want to put a little more info about what we mean by this here? i.e they are the ones who *lead* development of the thing, that we are always looking for qualified people to take on packages / submodules etc?

One of the original goals of the subpackage table was to try and motivate people with "ownership" clearly that didn't work too well.

File: project/roles.rst [hunk 4]
```diff
@@ -96,23 +96,21 @@ This primarily involves:
...
+----------
 
-Responsibilities of sub-package maintainers include:
+Responsibilities of a maintainer (subpackage, or library etc) include:
 
-* Evaluating and soliciting new pull requests which are consistent with the sub-package scope and contribution standards.
+* Evaluating and soliciting new pull requests which are consistent with their scope and contribution standards.
 * Providing material support for open pull requests to enable merging.
-* Maintaining and developing the sub-package roadmap consistent with the roadmap of the overall core package.
+* Maintaining and developing the roadmap consistent with the roadmap of the overall core package.
```
Comment (Cadair) [inline project/roles.rst:108]:
```suggestion
* Maintaining and developing the roadmap consistent with the roadmap of the project.
```

File: project/roles.rst [hunk 5]
```diff
@@ -96,23 +96,21 @@ This primarily involves:
...
-
-Responsibilities of documentation maintainers include:
-
-* Overseeing and improving content.
-* Soliciting and implementing improvements and new additions to the content.
-* Maintaining and improving the documentation infrastructure.
+* Responding to relevant queries on all official SunPy communication channels.
+* Handling GitHub issues associated with their scope.
+* Overseeing and improving documentation content.
+* Soliciting and implementing improvements and new additions to the documentation content.
```
Comment (Cadair) [inline project/roles.rst:113]:
```suggestion
* Soliciting and implementing improvements and new additions to related documentation content.
```

File: project/roles.rst [hunk 6]
```diff
@@ -96,23 +96,21 @@ This primarily involves:
...
 * Mentoring the next generation of maintainers and developers.
-
-Responsibilities of documentation maintainers include:
-
-* Overseeing and improving content.
-* Soliciting and implementing improvements and new additions to the content.
-* Maintaining and improving the documentation infrastructure.
+* Responding to relevant queries on all official SunPy communication channels.
+* Handling GitHub issues associated with their scope.
+* Overseeing and improving documentation content.
```
Comment (Cadair) [inline project/roles.rst:112]:
```suggestion
* Overseeing and improving related documentation content.
```

File: project/index.rst (hunk 2 above)
Comment (nabobalis) [reply project/index.rst]: Who are those people? I don't know.

File: project/index.rst (hunk 3 above)
Comment (nabobalis) [reply project/index.rst:214]: I can try.

**Comment by nabobalis** [comment]
This is good enough for now.

**Pull request merged**